services/search/pkg/search/provider/searchprovider.go

```diff
@@ -181,6 +181,9 @@ func (p *Provider) Search(ctx context.Context, req *searchsvc.SearchRequest) (*s
 			rootName = space.GetRootInfo().GetPath()
 			permissions = space.GetRootInfo().GetPermissionSet()
 			p.logger.Debug().Interface("grantSpace", space).Interface("mountpointRootId", mountpointRootID).Msg("searching a grant")
+		case "personal":
+			permissions = space.GetRootInfo().GetPermissionSet()
+			fmt.Println(permissions)
```

Comment:
You probably forgot to remove this Println

Reply:
ofc. Thanks!